helm/hpcc/values.schema.json

```diff
@@ -140,6 +140,44 @@
         ]
       }
     },
+    "dfuserver": {
+      "description": "dfuserver process",
+      "type": "array",
+      "items": {
+        "allOf": [
+          {
+            "name": {
+              "type": "string",
+              "description": "The name of the dfuserver process"
+            },
+            "disabled": {
+              "type": "boolean"
+            },
+            "enableSNMP": {
+              "type": "boolean",
+              "description": ""
+            },
+            "enableSysLog": {
+              "description": "",
+              "type": "boolean"
+            },
+            "queue": {
+              "description": "",
+              "type": "string"
+            },
+            "monitorqueue": {
+              "description": "",
+              "type": "string"
```

Comment:
does anyone know if and how monitorqueue works?
A quick scan suggests it's all based on scanning dfu workunits who have state DFUstate_monitoring, but I don't see anything that sets that.

@ghalliday @AttilaVamos ?